include/swift/Parse/Parser.h

```diff
@@ -690,6 +690,13 @@ class Parser {
     return Context.LangOpts.EnableExperimentalConcurrency;
   }
 
+  /// Returns true if a a Swift declaration starts after the current token, otherwise returns false.
```

Comment:
```suggestion
  /// Returns true if a Swift declaration starts after the current token, otherwise returns false.
```

Reply:
Also don't forget to run `git clang-format`, this line looks longer than the column limit

Reply:
Fixed and ran `git-clang-format`, can you please recheck? 🙏🏼